src/components/textfieldSlider/js/TextfieldSlider.js

```diff
@@ -113,65 +113,65 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
     fluid.defaults("fluid.slider", {
```

Comment:
Got it - the key I was missing was to have makeGradeLinkage target the same namespace, so that it's overridable by the end user. Thanks!